packages/lodestar/src/network/metadata.ts

```diff
@@ -103,3 +97,20 @@ export class MetadataController {
     return getENRForkID(this.config, this.chain.forkDigestContext, headEpoch);
   }
 }
+
+export function getENRForkID(
+  config: IBeaconConfig,
+  forkDigestContext: IForkDigestContext,
+  epoch: Epoch
+): phase0.ENRForkID {
+  const {currentFork, nextFork} = getCurrentAndNextFork(config, epoch);
+
+  return {
+    // Current fork digest
+    forkDigest: forkDigestContext.forkName2ForkDigest(currentFork.name),
+    // next planned fork versin
+    nextForkVersion: nextFork ? nextFork.version : currentFork.version,
+    // next fork epoch
+    nextForkEpoch: nextFork ? nextFork.epoch : FAR_FUTURE_EPOCH,
+  };
+}
```

Comment:
This util is only used by network/metadata so I've un-nested the metadata directory to the network root for simplicity.